### PR TITLE
Copybooks autocomplete introduction

### PR DESCRIPTION
**Pull request by abdrabrcom**
as per issue #106 introducing copybook list in autocomplete, at this point copybook names are being populated through usage in the COBOL file.

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -18,8 +18,14 @@
...
-		"mainframe", "lsp", "cobol", "z/os", "zos", "language-server", "jcl"
-	],
+        "mainframe",
+        "lsp",
+        "cobol",
+        "z/os",
+        "zos",
+        "language-server",
+        "jcl"
+    ],
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/package.json]: These changes are not part of this activity

File: clients/cobol-lsp-vscode-extension/package-lock.json [hunk 2]
```diff
@@ -2281,9 +2281,9 @@
 			"dev": true
 		},
 		"handlebars": {
-			"version": "4.4.3",
-			"resolved": "https://registry.npmjs.org/handlebars/-/handlebars-4.4.3.tgz",
-			"integrity": "sha512-B0W4A2U1ww3q7VVthTKfh+epHx+q4mCt6iK+zEAzbMBpWQAwxCeKxEGpj/1oQTpzPXDNSOG7hmG14TsISH50yw==",
+			"version": "4.7.3",
+			"resolved": "https://registry.npmjs.org/handlebars/-/handlebars-4.7.3.tgz",
+			"integrity": "sha512-SRGwSYuNfx8DwHD/6InAPzD6RgeruWLT+B8e8a7gGs8FWgHzlExpTFMEq2IA6QpAfOClpKHy6+8IqTjeBCu6Kg==",
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/package-lock.json]: These changes are not part of this activity

File: com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/completions/CopybookCompletionTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+          + "       PROCEDURE DIVISION.\n"
+          + "       PROGB.\n"
+          + "           PERFORM WITH TEST BEFORE UNTIL TBPARM1 = 0\n"
+          + "             SUBTRACT 1 FROM T\n"
+          + "             CALL 'ATCDEM4' \n"
+          + "           END-PERFORM\n"
+          + "       END PROGRAM ID1.";
+
+  @Test
+  public void testVariableCompletion() {
```
Comment (temanbrcom) [inline com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/completions/CopybookCompletionTest.java]: testCopybookCompletion?

File: com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/broadcom/lsp/cdi/module/service/ServiceModule.java [hunk 4]
```diff
@@ -66,6 +66,7 @@ private void bindCompletions() {
     completionBinding.addBinding().to(ParagraphCompletion.class);
     completionBinding.addBinding().to(SnippetCompletion.class);
     completionBinding.addBinding().to(KeywordCompletion.class);
+    completionBinding.addBinding().to(CopybookCompletion.class);
```
Comment (temanbrcom) [inline com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/broadcom/lsp/cdi/module/service/ServiceModule.java]: This should be duplicated in the TestModule

File: com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/completions/CopybookCompletionTest.java (hunk 3 above)
Comment (abdrabrcom) [reply com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/completions/CopybookCompletionTest.java]: fixed

File: com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/broadcom/lsp/cdi/module/service/ServiceModule.java (hunk 4 above)
Comment (abdrabrcom) [reply com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/broadcom/lsp/cdi/module/service/ServiceModule.java]: Added

File: clients/cobol-lsp-vscode-extension/package-lock.json (hunk 2 above)
Comment (abdrabrcom) [reply clients/cobol-lsp-vscode-extension/package-lock.json]: changes reverted

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (abdrabrcom) [reply clients/cobol-lsp-vscode-extension/package.json]: changes reverted

**Pull request merged**